src/QuadForm/Herm/Lattices.jl

```diff
@@ -359,7 +359,7 @@ function jordan_decomposition(L::HermLat, p)
 
   k = 1
   while k <= n
-    G = S * F * transpose(_map(S, aut))
+    G = S * F * transpose(map(aut, S))
```

Comment:
What did `_map` do?

Reply:
The same, but I run several examples using both functions and the underscore one seems to be slightly faster, allocations are the same... I will revert those changed. I first thought that this was a work-around before `map` was fully implemented.